data.sql

```diff
@@ -4,3 +4,9 @@ INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Gabumon', TO_DATE('15/11/2018', 'DD/MM/YYYY'), 8.0, true, 2);
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Pikachu', TO_DATE('07/01/2021', 'DD/MM/YYYY'), 15.04, false, 1);
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Devimon', TO_DATE('12/05/2017', 'DD/MM/YYYY'), 11.0, true, 5);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Charmander', TO_DATE('08/02/2020', 'DD/MM/YYYY'), -11.0, false, 0);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Plantmon', TO_DATE('15/09/2022', 'DD/MM/YYYY'), -5.7, true, 2);
```

Comment:
- [ ] Let's follow the guidelines and update the `Plantmon` date of birth.
<img width="782" alt="Capture" src="https://user-images.githubusercontent.com/58504655/135260721-472a188a-6ed4-4168-a139-5ad30f1a87d0.PNG">

Reply:
thanks for your feedback